pymtl3/passes/autotick/OpenLoopCLPass.py

```diff
@@ -20,27 +20,29 @@
 
 from ..BasePass import BasePass, PassMetadata
 from ..errors import PassOrderError
+from ..sim.PrepareSimPass import PrepareSimPass
 from ..sim.SimpleSchedulePass import SimpleSchedulePass, dump_dag
 from ..sim.SimpleTickPass import SimpleTickPass
 from ..tracing.CLLineTracePass import CLLineTracePass
-from ..tracing.CollectSignalPass import CollectSignalPass
-from ..tracing.PrintWavePass import PrintWavePass
+from ..tracing.PrintTextWavePass import PrintTextWavePass
 from ..tracing.VcdGenerationPass import VcdGenerationPass
 
-random.seed(0xdeadbeef)
 
 class OpenLoopCLPass( BasePass ):
   def __init__( self, print_line_trace=True ):
-    self.line_trace_on = print_line_trace
+    self.print_line_trace = print_line_trace
 
   def __call__( self, top ):
     if not hasattr( top._dag, "all_constraints" ):
       raise PassOrderError( "all_constraints" )
 
     top._sched = PassMetadata()
```

Comment:
Just leave this as a TODO: i think our goal is still to get rid of the `PassMetadata` and turn to use metadata APIs? One possible way to do this is to have all passes that have to communicate through the `_sched` and `_sim` namespace inherit from a dummy base pass that defines the necessary metadata. See https://github.com/pymtl/pymtl3/blob/a3d5a512d57969e1f37f1eecbb0c8e438079c88b/pymtl3/passes/rtlir/behavioral/BehavioralRTLIRGenL1Pass.py#L45

Reply:
Yes we will transition to APIs for sim and such. It's just that for _sched and _sim I haven't decided if we need to use a bass schedule pass or something else to store common metadata keys ...